### DISCUSSION TITLE
问题，授权账户无效，无法登录专业版

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#856] [Q&A]
账号：firekydchan@gmail.com

已购买专业版，通过6位device linking code登录，

电脑端：过一段时间就会自动变回免费版，多次登录，重启程序、重启电脑均无效；
手机端：可正常登录，但无法授权电脑端

![微信截图_20240911205408](https://github.com/user-attachments/assets/57c4b5eb-984f-4cb5-bc78-394440409700)

request failed with status code 503

ticket号码：76791388，logs ：
[lantern.log](https://github.com/user-attachments/files/16965235/lantern.log)

希望能帮忙解决，感谢 @Mike777Khoo

**Comment by XINYUAN1986** [comment]
我的也是这个问题，专业版无法登录。

**Comment by kidokogills** [comment]
> 账号：[firekydchan@gmail.com](mailto:firekydchan@gmail.com)
> 
> 已购买专业版，通过6位device linking code登录，
> 
> 电脑端：过一段时间就会自动变回免费版，多次登录，重启程序、重启电脑均无效； 手机端：可正常登录，但无法授权电脑端
> 
> ![微信截图_20240911205408](https://private-user-images.githubusercontent.com/157727057/366505787-57c4b5eb-984f-4cb5-bc78-394440409700.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMzE3NTUsIm5iZiI6MTcyNjEzMTQ1NSwicGF0aCI6Ii8xNTc3MjcwNTcvMzY2NTA1Nzg3LTU3YzRiNWViLTk4NGYtNGNiNS1iYzc4LTM5NDQ0MDQwOTcwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQwODU3MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOGZkYTUwOWI1OWJiZmJjNTYyMDg0NTIwNzg1NjViMjJhZjJjMGZiYzMyY2Y3NzNhZjcxYjUxMTEyOWRkMjc1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.z_5ht8G-OPyCshRhI4o8Dre01tQP-kpa_V0CPB9zTPU)
> 
> request failed with status code 503
> 
> ticket号码：76791388，logs ： [lantern.log](https://github.com/user-attachments/files/16965235/lantern.log)
> 
> 希望能帮忙解决，感谢 @Mike777Khoo

版本：7.9.5 (20240823.185949)

**Comment by XINYUAN1986** [comment]
*你好，你是WIN10？WIN7？还是MAC系统？我下载的WIN7版本，安装以后是7.8.2版本，跟你一样的问题。*

*但是我下载的MAC版本，在MAC系统里安装后显示是7.9.5版本，就不存在专业版无法登录的问题。*

kidokogills ***@***.***> 于2024年9月12日周四 16:58写道：

> ***@***.***
>
> 已购买专业版，通过6位device linking code登录，
>
> 电脑端：过一段时间就会自动变回免费版，多次登录，重启程序、重启电脑均无效； 手机端：可正常登录，但无法授权电脑端
>
> [image: 微信截图_20240911205408]
> <https://private-user-images.githubusercontent.com/157727057/366505787-57c4b5eb-984f-4cb5-bc78-394440409700.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMzE3NTUsIm5iZiI6MTcyNjEzMTQ1NSwicGF0aCI6Ii8xNTc3MjcwNTcvMzY2NTA1Nzg3LTU3YzRiNWViLTk4NGYtNGNiNS1iYzc4LTM5NDQ0MDQwOTcwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQwODU3MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOGZkYTUwOWI1OWJiZmJjNTYyMDg0NTIwNzg1NjViMjJhZjJjMGZiYzMyY2Y3NzNhZjcxYjUxMTEyOWRkMjc1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.z_5ht8G-OPyCshRhI4o8Dre01tQP-kpa_V0CPB9zTPU>
>
> request failed with status code 503
>
> ticket号码：76791388，logs ： lantern.log
> <https://github.com/user-attachments/files/16965235/lantern.log>
>
> 希望能帮忙解决，感谢 @Mike777Khoo <https://github.com/Mike777Khoo>
>
> 版本：7.9.5 (20240823.185949)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/856#discussioncomment-10623810>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLGIFBAYZ5AM6E4R7ZR7EF3ZWFJVBAVCNFSM6AAAAABOBEVQZWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRSGM4DCMA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>


-- 
silence ART

**Reply by kidokogills** [reply]
WIN11，7.9.5

**Comment by MaxMKMax** [comment]
@kidokogills 您好. 这是您的验证码. 您看行吗. 谢谢您的耐心等待.
767778

**Reply by kidokogills** [reply]
@Mike777Khoo 試過了，不行，而且已經不只是郵箱收不到驗證碼的問題了，是輸入驗證碼后回到主頁，依然是免費版，PC端跟手機都是這種情況，輸入驗證碼后顯示錯誤代碼500
